grudge/dof_desc.py

```diff
@@ -130,15 +147,15 @@ class DOFDesc:
 
     .. automethod:: uses_quadrature
 
-    .. automethod:: with_qtag
+    .. automethod:: with_discr_tag
     .. automethod:: with_dtag
 
     .. automethod:: __eq__
     .. automethod:: __ne__
     .. automethod:: __hash__
     """
 
-    def __init__(self, domain_tag, quadrature_tag=None):
+    def __init__(self, domain_tag, discretization_tag=None):
```

Comment:
```suggestion
    def __init__(self, domain_tag, discretization_tag=None, quadrature_tag=None):
```
Maybe keep it for backwards compatibility, in case someone called with the keyword? This would also need some more work in the constructor to warn.

Reply:
Done in https://github.com/inducer/grudge/pull/83/commits/4ed7e6f15401b6cb5da85c84f45ec267dbc2db1f!